packages/reapit-connect/serverless.yml

```diff
@@ -0,0 +1,111 @@
+# https://www.npmjs.com/package/serverless-single-page-app-plugin
+service: reapit-connect
+
+plugins:
+  - serverless-single-page-app-plugin
+  - serverless-plugin-ifelse
+
+custom:
+  s3Bucket: reapit-reapit-connect-${self:provider.stage}
```

Comment:
should we use reapit-connect instead of reapit-reapit-connect

Reply:
I generated this file using the cli in `utils`.
`reapit-connect` is the app name and `reapit-` is the prefix. So I think we should leave it like this